man/systemd.exec.xml

```diff
@@ -2911,6 +2911,20 @@ StandardInputData=V2XigLJyZSBubyBzdHJhbmdlcnMgdG8gbG92ZQpZb3Uga25vdyB0aGUgcnVsZX
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>LogIncludeRegex=</varname></term>
+        <term><varname>LogExcludeRegex=</varname></term>
+
+        <listitem><para>Define regexes used to filter log messages before they are logged to the journal. If
+        <varname>LogIncludeRegex=</varname> is defined, only logs that match against the regex will be written
+        to the journal. If <varname>LogExcludeRegex=</varname> is defined, only logs that do not match against
+        the regex will be written to the journal. Both <varname>LogIncludeRegex=</varname> and
+        <varname>LogExcludeRegex=</varname> can be used simultaneously. In this scenario, only log messages that
+        match against <varname>LogIncludeRegex=</varname> but not <varname>LogExcludeRegex=</varname> will be
+        written to the journal. If neither <varname>LogIncludeRegex=</varname> nor <varname>LogExcludeRegex=</varname>
+        are defined, all log messages are written to the journal.</para></listitem>
```

Comment:
please also add a comment saying explicitly that the regex is applied to the MESSAGE= field of structured messages, and none of the other fields.